user/serializers.py

```diff
@@ -1 +1,29 @@
-# write your code here
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = get_user_model()
+        fields = (
+            "id",
+            "username",
+            "email",
+            "password",
+            "is_staff",
```

Comment:
The `is_staff` field is included in the `fields` list but is marked as `read_only`. Since this field is not intended to be modified through the serializer, consider removing it from the `fields` list to simplify the serializer.